daemon/main.c

```diff
@@ -1080,6 +1097,12 @@ int main(int argc, char **argv) {
         log_init();
         error_log_limit_unlimited();
 
+        // --------------------------------------------------------------------
+        // get the certificate and start security
+#ifdef ENABLE_HTTPS
+        security_init();
```

Comment:
There are no return values and/or log messages to tell us if it succeeded or not.

Reply:
The security_init function is responsible to set the global variables and to start the openssl library, but before the version 1.1.0 all the functions except SSL_Library_init were void, and the SSL_Library_init always returns 1 according official documentation. For newest versions they began to deliver error, so I changed to have message.